src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -741,6 +745,7 @@ type FrameworkIdentifier =
         | DotNetFramework FrameworkVersion.V4_7_2 -> [ DotNetFramework FrameworkVersion.V4_7_1 ]
         | DotNetFramework FrameworkVersion.V4_8 -> [ DotNetFramework FrameworkVersion.V4_7_2 ]
         | DotNetFramework FrameworkVersion.V5 -> [ DotNetCoreApp DotNetCoreAppVersion.V3_1; DotNetStandard DotNetStandardVersion.V2_1 ]
+        | DotNetFramework FrameworkVersion.V6 -> [ DotNetCoreApp DotNetCoreAppVersion.V3_1; DotNetStandard DotNetStandardVersion.V2_1 ]
         | DotNet5WithOs Net5Os.Android -> [ DotNetFramework FrameworkVersion.V5; MonoAndroid MonoAndroidVersion.V10 ]
```

Comment:
In addition, net6.0 has OS specific TFMS like net5.0 does, it net6.0-android, the full list can be found here: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#net-5-os-specific-tfms

I think practically speaking you'd need to add a DotNet6WithOS like .net 5.0 has.

Reply:
thanks @baronfel, giving it a shot, is it worth to have some constructs for NetWindowsVersion and NetOsVersion that would take the dotnet version and have methods to check if the dotnet version is supported?